### PR TITLE
mockawesome

### PR DESCRIPTION
**Pull request by JulianO911**
image

### CODE REVIEW

**Review by NicolasB2 — COMMENTED** [review]
Could you upload an screenshot of HTML report?

File: package.json [hunk 1]
```diff
@@ -7,7 +7,11 @@
     "test:open": "npm run lint && cypress open",
     "test": "cypress run",
     "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports",
+    "pretest": "rm -r cypress/reports/*",
+    "posttest": "npm run generate:reports && combine:reports"
```
Comment (kliver98) [inline package.json]: this is not the order of execution

File: package.json [hunk 2]
```diff
@@ -7,7 +7,11 @@
     "test:open": "npm run lint && cypress open",
     "test": "cypress run",
     "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports",
+    "pretest": "rm -r cypress/reports/*",
```
Comment (kliver98) [inline package.json]: pretest should be improved, read more here https://ss64.com/bash/rm.html some flag that might help you if the folder/file does not exist

File: package.json [hunk 3]
```diff
@@ -5,9 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test:open": "npm run lint && cypress open",
+    "pretest": "rm -f cypress/reports/*",
     "test": "cypress run",
```
Comment (manuelq12) [inline package.json]: Well, you need to use the scripts "pretest" and "posttest" you created to generate the reports

You need to create a pipeline in test and test:open scripts

Comment (JulianO911) [reply package.json]: i did what you told me and the postest script throws me this 
image

**Review by manuelq12 — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -4,11 +4,11 @@
...
   "main": "index.js",
   "scripts": {
-    "clean:reports": "rm -f cypress//reports",
-    "test:open": "npm run lint && npm run pretest && cypress open || npm run posttest",
+    "clean:reports": "rm -f cypress/reports",
+    "test:open": "npm run lint && cypress open",
     "pretest": "npm run clean:reports",
-    "test": "cypress run",
-    "posttest": "npm run combine:reports && generate:reports",
+    "test": "cypress run || npm run posttest ",
```
Comment (holgiosalos) [inline package.json:10]: I understand with this, that you want to generate the report even if the test fails, if so, you can change the command in this way: `cypress run || true` the ` || true` changes the exit command of the test, so the posttest will be always executed. However, this is unorthodox, I don't know the smart way

**Pull request merged**